docker/services/uwsgi/uwsgi.d/40_log.ini

```diff
@@ -1,5 +1,4 @@
 [uwsgi]
 log-format = [uwsgi] %(var.HTTP_X_FORWARDED_FOR) (%(addr)) - - [%(ltime)] "%(method) %(uri) %(proto)" %(status) %(size) "%(referer)" "%(uagent)" host_hdr=%(host) req_time_elapsed=%(msecs) process=%(pid) worker=%(wid)
-logfile-chmod = 644
```

Comment:
This configuration option isn't used when using `logger`, its only used when using the `log-to` configuration. So we might as well drop it, since its deceiving that it is here.

When we switched to using `logger` in this config is probably when it broke.